docs/get-started/introduction.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Quick start"
+layout: default
+sort: 1
+---
+
+# Introduction
+
+The Node Feature Discovery Operator manages the detection
+of hardware features and configuration in an OpenShift
+cluster by labeling the nodes with hardware-specific information.
+The Node Feature Discovery will label the host with
+node-specific attributes,
+like PCI cards, kernel, or OS version, and many more.
+
+Learn about the Node Feature Discovery Operator and
+how you can use it to expose node-level
+information by orchestrating the Node Feature Discovery,
+a Kubernetes add-on for detecting hardware features and system configuration!
```

Comment:
This paragraph somehow feels disconnected/random 🤔  Learn where, how? Maybe extend the text/idea a bit, add links or smth?